admin/code/LeftAndMain.php

```diff
@@ -683,14 +683,40 @@ public function Link($action = null) {
 	}
 
 	/**
-	 * Returns the menu title for the given LeftAndMain subclass.
-	 * Implemented static so that we can get this value without instantiating an object.
-	 * Menu title is *not* internationalised.
+	 * @deprecated 5.0
 	 */
 	public static function menu_title_for_class($class) {
+		Deprecation::notice('5.0', 'Use menu_title() instead');
+		return static::menu_title($class, false);
+	}
+
+	/**
+	 * Get menu title for this section (translated)
+	 *
+	 * @param string $class Optional class name if called on LeftAndMain directly
+	 * @param bool $localise Determine if menu title should be localised via i18n.
+	 * @return string Menu title for the given class
+	 */
+	public static function menu_title($class = null, $localise = true) {
+		if($class && is_subclass_of($class, __CLASS__)) {
+			// Respect oveloading of menu_title() in subclasses
+			return $class::menu_title(null, $localise);
+		}
+		if(!$class) {
+			$class = get_called_class();
```

Comment:
> I still think its a weird mix of using a static method as an instance method

It is weird, but it is a valid use case... it's kind of like negative margins. :D